README.md

```diff
@@ -60,72 +56,13 @@ If you'd like to try it out or have any questions - feel free to join our [Slack
 # Try out Opta
 
 
-To use Opta, you first need to create some simple yaml configuration files that describe your needs. You can use 
-our [magical UI](https://app.runx.dev/yaml-generator) to help generate these files or do it manually (described below).
-
-### Four Step Quick Start (<30min)
```

Comment:
Can you make sure the getting started in readme is "good"?

Reply:
copied the getting started from here to our docs